app/components/round-form.js

```diff
@@ -59,6 +86,20 @@ export default Ember.Component.extend({
     save() {
       this.get('model').save()
       .then((round) => this.sendAction('didSave', round));
+    },
+
+    closeFriendsDialog(event) {
+      if(event === 'save') {
+        // do ssomething
```

Comment:
Placeholder to make sure things worked like I wanted. Will implement when the "who are you playing with" dialog is implemented